neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Tady ta exception už neni potřeba, když se chytá v `build_model`, ne? Jediný, v čem to tady může shořet je když se něco stane při načítání (z toho se beztak nedá vzpamatovat), anebo když se vyhodí ta exception z řádku 49 a s tou se taky nic dělat nedá. Takže celej tenhle try-catch blok nemá smysl.